include/roboteam_ai/TreeHelper/RobotOutOfFieldHelper.h

```diff
@@ -0,0 +1,18 @@
+//
+// Created by jessevw on 21.11.19.
+//
+
+#ifndef RTT_ROBOTOUTOFFIELDHELPER_H
+#define RTT_ROBOTOUTOFFIELDHELPER_H
+#include <include/roboteam_ai/bt/Node.hpp>
```

Comment:
It's minor but please be consistent with include style between doing include/ or doing bt/ and the usage of ""and <> between different files of the same "type" (such as these helpers).